spec/PeriodicBackgroundSync-index.bs

```diff
@@ -114,21 +114,25 @@ In response to these triggers, the scheduler either schedules delayed processing
 
 The <dfn>effective minimum sync interval for origin</dfn> |origin| (an [=origin=]), is the [=minimum periodic sync interval for any origin=] + some user agent defined amount for the |origin|.
 
-Note: The user agent defined amount could be based off the amount of engagement the user has with the origin. This value can be different each time [=effective minimum sync interval for origin=] is called.
+Note: The user agent defined amount SHOULD be based off the amount of engagement the user has with the origin. This value can be different each time [=effective minimum sync interval for origin=] is called. This value can also be a user-agent defined INFINITE value, which effectively suspends firing any [=periodicsync event|periodicsync events=] until user engagement with the [=origin=] increases above a user agent defined threshold.
```

Comment:
I'm not sure you can use "should" in a note, since it's typically normative. Is bikeshed throwing an error/warning here?

Maybe the part about infinite should be in the normative section, so:

The <dfn>effective minimum sync interval for origin</dfn> |origin| (an [=origin=]), is the [=minimum periodic sync interval for any origin=] + some user agent defined amount for the |origin| (which may be `Infinity`). The user agent defined amount must be used to meet the requirements recommended in [[#privacy]], [[#security]] and [[#resources]].

Note: The user agent defined amount can be based on the amount of engagement the user has with the origin. An infinite value effectively suspends firing any [=periodicsync event|periodicsync events=]. This value can be different each time [=effective minimum sync interval for origin=] is called.

Reply:
No, bikeshed isn't throwing an error. I've updated per your recommendation anyway.